websites/H/HentaiVN/presence.ts

```diff
@@ -0,0 +1,499 @@
+				103: "Mother",
+				205: "Nakadashi",
+				1: "Netori",
+				201: "Non-hen",
+				68: "NTR",
+				272: "Nun",
+				69: "Nurse",
+				211: "Old Man",
+				71: "Oneshot",
+				70: "Oral",
+				209: "Osananajimi",
+				72: "Paizuri",
+				204: "Pantyhose",
+				276: "Ponytail",
+				73: "Pregnant",
+				98: "Rape",
+				258: "Rimjob",
+				117: "Romance",
+				207: "Ryona",
+				134: "Scat",
+				74: "School Uniform",
+				75: "SchoolGir",
+				87: "Series",
+				88: "Sex Toys",
+				246: "Shimapan",
+				118: "Short Hentai",
+				77: "Shota",
+				76: "Shoujo",
+				79: "Siscon",
+				78: "Sister",
+				82: "Slave",
+				213: "Sleeping",
+				84: "Small Boobs",
+				278: "Son",
+				83: "Sports",
+				81: "Stockings",
+				85: "Supernatural",
+				250: "Sweating",
+				86: "Swimsuit",
+				266: "Tall Girl",
+				91: "Teacher",
+				89: "Tentacles",
+				109: "Time Stop",
+				90: "Tomboy",
+				252: "Tracksuit",
+				256: "Transformation",
+				92: "Trap",
+				274: "Truyện Việt",
+				111: "Tsundere",
+				93: "Twins",
+				261: "Twintails",
+				107: "Vampire",
+				208: "Vanilla",
+				95: "Virgin",
+				270: "Webtoon",
+				94: "X-ray",
+				112: "Yandere",
+				96: "Yaoi",
+				97: "Yuri",
+				128: "Zombie"
+			};
+			keystr = keystr
+				.toString()
+				.replace(
+					/\b(?:3|5|116|203|20|21|249|131|127|22|279|277|115|130|257|251|24|25|133|23|32|267|27|28|29|30|254|33|248|31|242|241|39|101|129|34|35|271|100|36|120|210|38|2|275|269|40|262|268|132|212|104|105|253|41|260|42|44|4|43|45|245|125|46|123|243|47|48|108|259|37|202|50|51|206|52|106|56|264|53|55|247|57|58|102|80|122|59|60|61|244|62|26|280|99|110|265|63|64|273|114|65|119|124|126|121|113|263|66|67|103|205|1|201|68|272|69|211|71|70|209|72|204|276|73|98|258|117|207|134|74|75|87|88|246|118|77|76|79|78|82|213|84|278|83|81|85|250|86|266|91|89|109|90|252|256|92|274|111|93|261|107|208|95|270|94|112|96|97|128)\b/gi,
+					matched => keynum[matched]
+				);
+			presenceData.state = `Từ khóa: ${namekey} - Doujin: ${doukey} - Nhân vật: ${charkey} - Thể loại: ${keystr}`;
+		} else presenceData.state = "Đang nhập dữ liệu tìm kiếm...";
+	} else if (document.location.pathname.startsWith("/forum/nhan_tin.php")) {
+		presenceData.details = "Đang nhắn tin...";
+		presenceData.state = document.querySelector("h3").textContent.trim();
+		presenceData.smallImageKey = "chater";
+	} else if (document.location.pathname.startsWith("/forum/mail.php")) {
+		if (document.location.search.includes("page"))
+			page = new URLSearchParams(document.location.search).get("page");
+
+		presenceData.details = "Đang xem hộp thư...";
+		presenceData.state = `Trang ${page}`;
+		presenceData.smallImageKey = "chat";
+	} else if (document.location.pathname.startsWith("/forum/free-market.php")) {
+		presenceData.details = "Đang lướt chợ tự do...";
+		presenceData.smallImageKey = "shop";
+	} else if (document.location.pathname.startsWith("/forum/shop.php")) {
+		presenceData.details = "Đang lướt shop...";
```

Comment:
```suggestion
		presenceData.details = "Đang duyệt shop...";
```

Reply:
That is the term used in the website, which cannot be changed.